From fb0c54a70813278c6211feb500ca6472b0d98906 Mon Sep 17 00:00:00 2001
From: Daniel Borkmann <dborkman@redhat.com>
Date: Tue, 16 Sep 2014 15:04:39 -0400
Subject: [net] vxlan: remove extra newline after function definition

Message-id: <de19631a96485e84e10d5e3fb5205c1bc3b71e17.1410441647.git.dborkman@redhat.com>
Patchwork-id: 95085
O-Subject: [RHEL7.1 PATCH net 2/7] vxlan: remove extra newline after function definition
Bugzilla: 1130643
RH-Acked-by: David S. Miller <davem@redhat.com>
RH-Acked-by: Vlad Yasevich <vyasevic@redhat.com>
RH-Acked-by: Jiri Pirko <jpirko@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1130643
Upstream Status: 4fe46b9a4d0b5eef96867e6d5134159e5a65d2a5
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7947483
Tested: verified by IBM and myself

Upstream Commit:
commit 4fe46b9a4d0b5eef96867e6d5134159e5a65d2a5
Author: Daniel Baluta <dbaluta@ixiacom.com>
Date:   Fri Jan 31 09:50:12 2014 +0200

    vxlan: remove extra newline after function definition

    Signed-off-by: Daniel Baluta <dbaluta@ixiacom.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Daniel Borkmann <dborkman@redhat.com>
Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/vxlan.c b/drivers/net/vxlan.c
index 73674b9..c57fc24 100644
--- a/drivers/net/vxlan.c
+++ b/drivers/net/vxlan.c
@@ -462,7 +462,6 @@ static inline struct hlist_head *vxlan_fdb_head(struct vxlan_dev *vxlan,
 /* Look up Ethernet address in forwarding table */
 static struct vxlan_fdb *__vxlan_find_mac(struct vxlan_dev *vxlan,
      const u8 *mac)
-
 {
  struct hlist_head *head = vxlan_fdb_head(vxlan, mac);
  struct vxlan_fdb *f;
-- 
1.7.1